buildh

```diff
@@ -75,40 +124,39 @@ def command_makei(profile):
     _exec_remote_script('\n'.join(remote_script), profile['host'])
 
 
-def _build_cd_script(directory):
-    return "cd '{}'".format(directory)
+def _build_cd_script(directory: str) -> str:
+    return f"cd '{directory}'"
 
 
-def _build_make_script():
-    version = subprocess.check_output(['git', 'describe']).rstrip()
-    return 'make VERSION={}'.format(version)
+def _build_make_script() -> str:
+    version: str = subprocess.check_output(['git', 'describe'], text=True).rstrip()
+    return f'make VERSION={version}'
 
 
-def _build_rsync_script(host, directory):
-    return '''
-rsync -v -rlp \
+def _build_rsync_script(host: str, directory: str) -> str:
+    return f'''
+    rsync -v -rlp \
     --include '*.c' \
     --include '*.h' \
     --include /device \
     --include /Makefile \
     --exclude '*' \
-    . {host}:{directory}
-'''.format(host=host, directory=directory)
+    . root@{host}:{directory}
```

Comment:
I would not add a user without making it configurable. If someone wants to use `root` or `pcm` they can add the user to there `~/.ssh/config` with the old version